index.html

```diff
@@ -61,11 +66,12 @@ <h1 class="logo">My Logo</h1>
        <section class="parting">
         <div class="tonic">
           <div class="tonicing">
-            <img src="images/snapshot1.png" alt="Tonic Work" />
+            <img class="photo1" src="images/snapshot1.png" alt="Tonic Work" />
+            <img class="photo2" src="images/Snapshoot Portfolio.png" alt="Tonic Work" />
           </div>
           <div class="tonicContent">
-            <h2>Tonic</h2>
-            <div class="tonicList1">
+            <h2 class="text-4">Tonic</h2>
+            <div class="tonicListR">
               <ul>
                 <li class="licap">CANOPY</li>
                 <li>• Back End Dev</li>
```

Comment:
- [ ] [Optional]The page is divided into half of it containing the project's picture and the other half containing the project's description. please implement it
### Your design
![your](https://user-images.githubusercontent.com/81727988/194684642-e6e85896-aa90-474d-851b-0e656244e435.png)
### Figma design
![his](https://user-images.githubusercontent.com/81727988/194684649-8d8994b1-27e3-41f1-9839-f7561ba2cc68.png)